L/LoopTools/bundled/patches/simplify-configure.patch

```diff
@@ -58,18 +61,14 @@
 -  exit 1 ;;
 -esac
 -
-+# We know the compiler appends underscores
-+echo "yes" 1>&3
-+CONF_NOUNDERSCORE=0
-+  
  CONF_CFLAGS+=" $CONF_BITS"
  CONF_CXXFLAGS+=" $CONF_BITS"
  
-@@ -278,27 +259,14 @@
+@@ -276,29 +257,15 @@
  
- echo -n "does $CONF_FC support REAL*$CONF_QUADSIZE... " 1>&3
+ ## does f77 support REAL*16?
  
--rm -fr $test*
+-echo -n "does $CONF_FC support REAL*$CONF_QUADSIZE... " 1>&3
 +if [[ "${target}" == arm-* ]]; then
 +    # On 32-bit the Fortran compiler doesn't support quadsize reals
```

Comment:
For the record, the error we get for `aarch64-apple-darwin` is the same as the one we'd get for 32-bit ARM platforms without this setting, so including `aarch64-apple-darwin` here would do the trick.  However I'm wondering if updating the GCC toolchain for this platform would solve the issue, which is something I'd like to try anyway in the coming days.

Reply:
Updating the GCC toolchain for aarch64-apple-darwin fixed it, yay :partying_face: